src/TBuffer.cpp

```diff
@@ -51,529 +51,530 @@
 // clang-format off
```

Comment:
READ MY LIPS clang-format I TOLD YOU __NOT__ TO TOUCH THIS! 😡 What part of  `// clang-format off` and [ClangFormatStyleOptions](http://clang.llvm.org/docs/ClangFormatStyleOptions.html) can you not understand?